docker/dev/init_hook.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
```

Comment:
Script is missing old functionality provided by `RETAIN_SUPERVISOR` variable.

Reply:
True, however, hook scripts are started in a dedicated environment so `RETAIN_SUPERIVSOR` set to the docker environment would not be passed to the script anyway. We would need it supported in the supervisor plugin I guess.

But we can live without it now, didn't we? The hook script takes care of the termination of the init. Remove the hook from the `supervisor.conf` and we have basically the same functionality the `RETAIN_SUPERVISOR` provides. What do you think?